stdlib/array.gr

```diff
@@ -892,3 +895,49 @@ export let sort = (comp, array) => {
   let len = length(array)
   quicksort(0, len - 1)
 }
+
+/**
+ * Rotates an array by n elements to the right, in place.
+ *
+ * If n is negative, the array will be rotated by n elements
+ * to the left. See examples.
+ *
+ * @param arr: The array to be rotated
+ * @param n: The number of elements to rotate by
+ * @returns The rotated array
+ *
+ * @example: rotate([> 1, 2, 3, 4, 5], 2) == [> 4, 5, 1, 2, 3]
+ * @example: rotate([> 1, 2, 3, 4, 5], -1) == [> 2, 3, 4, 5, 1]
+ * @since v0.4.5
+ */
+export let rotate = (arr, n) => {
```

Comment:
```suggestion
export let rotate = (n, arr) => {
```
Swapping the array argument because we are making everything data-last right now (this might change in the future, but this is our blanket rule).